package.json

```diff
@@ -140,7 +140,11 @@
   "lint-staged": {
     "*.js": [
       "prettier --write",
+      "eslint --max-warnings=0",
```

Comment:
[note] I excluded `--cache` option to lint files in the clean state. I think it no performance problem because of not for all JS files in the project.